src/Redux.js

```diff
@@ -5,7 +5,7 @@ export default class Redux {
   constructor(dispatcher, initialState) {
     if (typeof dispatcher === 'object') {
       // A shortcut notation to use the default dispatcher
-      dispatcher = createDispatcher(composeStores(dispatcher));
+      dispatcher = createDispatcher(composeStores(dispatcher), initialState);
```

Comment:
Haha it'll change soon once we get to a stabler place, but in the meantime we don't want to be breaking the API for every new PR.